docs/how-to/enforce-permissions/data-filtering.mdx

```diff
@@ -0,0 +1,67 @@
+---
+sidebar_position: 4
+title: Data Filtering
+---
+
+Data filtering in permissions represents is a different kind of approach when enforcing permissions.
+Instead of merely granting or denying access, it curates what users see, tailoring the data to their individual permissions.
+This method ensures not only secure access but also optimized data delivery.
+
+## Simple usage
+
+```go
+package main
+
+import (
+	"fmt"
+)
+import p "github.com/permitio/permit-golang/pkg/permit"
```

Comment:
```suggestion
import "github.com/permitio/permit-golang/pkg/permit"
```

Reply:
That syntax is to alias permit package as p ( in python it would be `from x import permit as p` )
I do that to keep the permit. when calling the function ( duplicate name permit, package and variable holding the client )